langserver/handlers/execute_command_rootmodules.go

```diff
@@ -0,0 +1,73 @@
+package handlers
+
+import (
+	"context"
+	"fmt"
+	"sort"
+
+	"github.com/creachadair/jrpc2/code"
+	lsctx "github.com/hashicorp/terraform-ls/internal/context"
+	ilsp "github.com/hashicorp/terraform-ls/internal/lsp"
+	lsp "github.com/sourcegraph/go-lsp"
+)
+
+const rootmodulesCommandResponseVersion = 0
+const rootmodulesCommandURIArgNotFound code.Code = -32004
+
+type rootmodulesCommandResponse struct {
+	Version     int         `json:"version"`
```

Comment:
Nitpick: I'd probably call this `FormatVersion` instead, to avoid any confusion with any other version - such as Terraform or LS version.

Reply:
It's the version of the response, is `FormatVersion` a common name for that sort of attribute? (`FormatVersion` feels a bit misnomer to me, but I'm not an API expert, what about `ResponseVersion`)?

Reply:
`ResponseVersion` sounds good too 👍  , I was thinking of "format" in the context of "response format", mostly stealing that idea from `terraform-json` https://github.com/hashicorp/terraform-json/blob/44141229df28a1ed34857680562e42326fe6af57/schemas.go#L21